doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_nordic_bundles.rst

```diff
@@ -0,0 +1,57 @@
+
+These components are provided in the form of the so called "nRF54h20 SOC binaries", which can be found in https://files.nordicsemi.com/ui/repos/tree/General/SDSC
+
+Updating the Nordic Semiconductor firmware components
+*****************************************************
+
+Updating the Nordic Semiconductor firmware components can be done in two ways.
+Both require using the nordic_top.suit envelope, which can be found inside the Nordic bundle .zip file.
+These two methods are:
+
+* Updating it separately from the manufacturer application
+* Updating it along with the manufacturer application, by attaching the nordic_top.suit envelope to the manufacturer root manifest
+
+Updating the Nordic components separately from the manufacturer application
+===========================================================================
+
+Updating the Nordic components separately from the manufacturer application is done by performing two separate updates.
+One is done with a manufacturer envelope with the update candidate.
+The second one is performed by providing the nordic_top.suit envelope found inside the Nordic Bundle to the device.
+The envelopes contain all the information allowing the device to differentiate which of the updates is performed.
+
+The benefits of such an approach are:
+
+* A smaller partition for storing the update candidate is required.
+* There is no need to integrate the process of downloading Nordic artitfacts into the manufacturer application building process.
+
+The drawbacks are:
+
+* Two separate updates are required, which is not supported by all protocols.
+* The manufacturer envelope has no possibility to ensure the compatibility of the Nordic components with the manufacturer application.
+
+
+Attaching the nordic_top.suit envelope to the manufacturer root manifest
+========================================================================
+
+When building an application, the build system can be configured to include the Nordic components inside the SUIT root envelope used for updates.
+
+To do this:
+
+1. Unpack the Nordic bundle .zip file to a directory of your choice.
+   One of the extracted files will be the nordic_top.suit envelope.
+
+2. Build the application, pointing to the directory containing the nordic_top.suit envelope.
+   This is done by setting the `SUIT_ENVELOPE_NORDIC_TOP_DIRECTORY` sysbuild configuration option.
+   For example:
+
+   ``west build -b nrf54h20dk/nrf54h20/cpuapp --  -DSB_CONFIG_SUIT_ENVELOPE_NORDIC_TOP_DIRECTORY="\"<NORDIC_TOP_DIRECTORY>\""``
+
+   Replace `<NORDIC_TOP_DIRECTORY>` with the path to the directory containing the extracted nordic_top.suit file.
+
+3. Use the ``build/DFU/root.suit`` file generated by the build system to update both the manufacturer application and the Nordic Semiconductor firmware components.
```

Comment:
Please either add advantages/drawbacks here too or make a separate section that compares those two methods.

Reply:
Ok, at first I wanted to avoid duplication, but I didn't have an idea how to do a "comparison" in a nice way, so I simply copied the pros/cons, switched them, and reworded them in the second section